src/applications/dispute-debt/config/contactInfo.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+// import profileContactInfo from 'platform/forms-system/src/js/definitions/profileContactInfo';
+import { getContent } from 'platform/forms-system/src/js/utilities/data/profile';
+import profileContactInfo from 'applications/_mock-form-ae-design-patterns/shared/components/ContactInfo/profileContactInfo';
```

Comment:
We may need to steal the source components because of the cross app import rules. I wonder if we can reach out and see if there are plans to convert this to a platform managed component